scripts/download-methyl.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I think you need to merge main into the download script branch, then the download script into this branch to get this to not show up